src/base/Raw.cc

```diff
@@ -23,18 +23,22 @@ Raw::print(std::ostream &os) const
         return os;
 
     // finalize debugging level if no level was set explicitly via minLevel()
+    const auto printLimit = std::min(size_, printableSize_);
```

Comment:
We never print fewer (or more) bytes than this variable, so this variable is not really a "limit" but "size":


```suggestion
    const auto printableDataSize = std::min(size_, printableDataSizeLimit_);
```

Reply:
I do not insist on any specific name -- feel free to propose something better while still addressing my concern. For example, `reportableDataSize` and `reportableDataSizeLimit_` could also work (but I think "printable" is better here).

> this algorithm is not limited to "printable" bytes

The word "printable" has several interpretations. IMHO, the proposed name matches one of them, and humans would understand (with high probability) which meaning is used _in this specific context_.

> This is the limit bounding output of the action

It is not a limit; it is the _exact_ size -- we never send fewer data bytes to the printing code; we send exactly that many data bytes.

> The object itself may be larger ...
> [the caller may lie about the object size]

Yes, but those facts are not relevant to this particular variable.

> `asHex` feature means that we cannot consider this to be the "printable" size/amount of output either.

Yes, and that is why we should not call this variable "outputSize" or something like that. Suggested "printableDataSize" does _not_ imply that formatted/decorated output is going to have the same size.

Reply:
So, using your definitions described here to re-write the title and description.